.github/workflows/config.yml

```diff
@@ -3,7 +3,7 @@ on: [push]
 jobs:
   install-gemmini:
     name: gemmini-install
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
You should not run the same `conda init` and other things in https://github.com/ucb-bar/gemmini/blob/6f57972db9b0815462cc0569f922792f83e35c5d/.github/scripts/enable-conda.sh#L4-L12 since this affects overall `.bashrc` and might mess up the conda setup on the CI machine.